pkg/minikube/constants/constants_linux.go

```diff
@@ -18,7 +18,14 @@ limitations under the License.
 
 package constants
 
+import (
+	"k8s.io/client-go/util/homedir"
+)
+
 var SupportedVMDrivers = [...]string{
 	"virtualbox",
 	"kvm",
 }
+
+var DefaultMountDir = homedir.HomeDir()
```

Comment:
Ah makes sense. It's still a little weird to use the client-go import here. Can we use the std lib homedir?